WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -7128,7 +7061,7 @@ function WeakAuras.CreateFrame()
     local usedIcons = {};
     local num = 0;
     if(subname ~= "") then
-      for name, icons in pairs(spellCache) do
+      for name, icons in pairs(WeakAuras.GetSpellCache()) do
```

Comment:
So, in the spirit of making the code oragnization better. I think the whole object should be WeakAuras.spellcache, with methods on it. e.g. WeakAuras.spellcache.Add(), WeakAuras.spellcache.Load(), WeakAuras.spellcache.Get().

And I think this whole matching code could then be e.g. WeakAuras.spellcache.bestMatch( key ).